content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Service Accounts
+description: >
+  Learn about the Kubernetes ServiceAccount object.
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts?
+
+A service account is a type of non-human account that, in Kubernetes, provides
+an identity for the processes that run in Pods. Pods can use service accounts
```

Comment:
processes in pods are the primary use, but are not the only one... service account credentials can be obtained and used outside of pods as well (notably, the kubernetes controller manager uses a distinct service account for every controller loop)

Reply:
Done

Reply:
Not sure how this was resolved yet - sorry. We can suggest wording if that'll help.

Reply:
would something like this work?

```suggestion
an identity for cluster processes, such as Pods and controllers, to use when talking to the API server
```

Reply:
> ```markdown
> A service account is a type of non-human account that, in Kubernetes, provides
> an identity for cluster processes, such as Pods and controllers, to use when talking to the API server
> to make calls to the Kubernetes API server. Service accounts exist as
> ```

It's not quite that. You can also use a ServiceAccount for things other than authenticating to the API server,
so we need to frame this authn piece as only one part of the picture.

For example, you can use a ServiceAccount to define which image pull secrets a Pod can use. ServiceAccounts are really there to define an identity (≅ authentication principal) for _sets_ of Pods. Those sets of Pods might be the same as an app component, or:
- one app might use several different ServiceAccounts, for different things
- several apps might all use the same ServiceAccount

Reply:
When Pods use a ServiceAccount, the Pods and the ServiceAccount have to be in the same namespace (this isn't implemented as a check, it's just - by design -  the only way the API lets you specify it).